modules/lib/src/main/resources/assets/lib/ckeditor/plugins/nbsp/plugin.js

```diff
@@ -15,7 +15,7 @@ CKEDITOR.plugins.add( 'nbsp',
 				editor.insertHtml( '&nbsp;', 'text' );
 			}
 		});
-		editor.setKeystroke( CKEDITOR.ALT + 32 /* space */, 'insertNbsp' );
+		editor.setKeystroke( CKEDITOR.CTRL + 32 /* space */, 'insertNbsp' );
```

Comment:
It is better to update HtmlEditor.ts and add there next code to the other keystrokes assignments:
```
this.editor.on('instanceReady', () => {
            this.editor.setKeystroke(CKEDITOR.CTRL + 70, 'find'); // open find dialog on CTRL + F
            this.editor.setKeystroke(CKEDITOR.CTRL + 75, 'link'); // open link dialog on CTRL + K
            this.editor.setKeystroke(CKEDITOR.CTRL + 76, 'image'); // open link dialog on CTRL + L
            this.editor.setKeystroke(CKEDITOR.CTRL + CKEDITOR.ALT + 49, 'h1'); // apply Heading 1 format
            this.editor.setKeystroke(CKEDITOR.CTRL + CKEDITOR.ALT + 50, 'h2'); // apply Heading 2 format
            this.editor.setKeystroke(CKEDITOR.CTRL + CKEDITOR.ALT + 51, 'h3'); // apply Heading 3 format
            this.editor.setKeystroke(CKEDITOR.CTRL + CKEDITOR.ALT + 52, 'h4'); // apply Heading 4 format
            this.editor.setKeystroke(CKEDITOR.CTRL + CKEDITOR.ALT + 53, 'h5'); // apply Heading 5 format
            this.editor.setKeystroke(CKEDITOR.CTRL + CKEDITOR.ALT + 54, 'h6'); // apply Heading 6 format
            this.editor.setKeystroke(CKEDITOR.CTRL + CKEDITOR.SHIFT + 55, 'p'); // apply the 'Normal' format
            this.editor.setKeystroke(CKEDITOR.CTRL + CKEDITOR.SHIFT + 56, 'div'); // apply the 'Normal (DIV)' format
            this.editor.setKeystroke(CKEDITOR.CTRL + CKEDITOR.SHIFT + 57, 'address'); // apply the 'Address' format
            **this.editor.setKeystroke( CKEDITOR.CTRL + 32 /* space */, 'insertNbsp' );**
        });
```

All plugins added since last update of CKEditor should be removed from `plugins` directory on next CKE update and included as a part of a CKE lib (after importing our `build-config.js` on the CKE website we will need to add those plugins to our custom build)